utils/mvn/utils.go

```diff
@@ -48,48 +50,90 @@ func RunMvn(configPath, deployableArtifactsFile string, buildConf *utils.BuildCo
 	return mvnRunConfig.runCmd()
 }
 
-func getMavenHome() (string, error) {
+func getMavenHomeAndValidateVersion() (string, error) {
```

Comment:
Could you please also write a unit test in utils_test.go for getMavenHomeAndValidateVersion?
It can run this function and expect a correct maven home path without error.